pages/linux/hello.md

```diff
@@ -0,0 +1,16 @@
+# hello
+
+> Print "Hello, world!", "hello, world" or a customizable text.
+> More information: <https://www.gnu.org/software/hello/>.
+
+- Print "Hello, world!":
+
+`hello`
+
+- Print "hello, world", the traditional type:
+
+`hello --traditional`
+
+- Print a custom text:
+
+`hello --greeting="{{tldr}}"`
```

Comment:
I don't think I like this parameter name. We usually try to give more descriptive names. I would recommend something like "Some greeting".

Reply:
Taking a cue from [echo](https://github.com/tldr-pages/tldr/blob/master/pages/common/echo.md), how about, `Print a text message` ?

Using "greeting.." might be ambiguous, I believe. (even though it's the argument itself :D )

Reply:
Yes that is great 👍

Reply:
Updated.

Reply:
I'm not seeing the update, @sahilister?

Reply:
Hello @sbrl , [here](https://github.com/tldr-pages/tldr/pull/4586/commits/bb1bda182083de3685ae09d4778329471f207ff8) it's showing changes from `Print a custom text` -> `Print a text message`.

A tiny change, maybe that's why it slipped. :)

Reply:
Oh I see @sahilister! Thanks for that. I think @schneiderl also meant to change the `tldr` message here too. What about this?
```suggestion
`hello --greeting="{{greeting_text}}"`
```

Reply:
aah! sorry I missed that one, thanks @sbrl